src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -455,6 +457,8 @@ void DeleteInvalidBlocks(Keccak invalidBlockHash)
                     options,
                     new GethLikeBlockTracer(GethTraceOptions.Default),
                     DumpOptions.Geth);
+                
+                OnInvalidBlock?.Invoke(this, ex);
```

Comment:
Can we keep to the .NET standard and use EventArgs descendant instead of Exception directly?

Reply:
Ok